src/Components/ContactForm.jsx

```diff
@@ -11,54 +11,152 @@ const useStyles = makeStyles({
+  const [toSend, setToSend] = useState({
+    from_name: "",
+    reply_to: "",
+    contact_type: "",
+    message: ""
+  })
 
-  return (
-    <Box className={classes.root}>
-      <form className={classes.formStyle}>
-        <label>Name</label>
-        <input
-          className={classes.inputStyle}
-          type="text"
-          name="name"
-          defaultValue="Enter Your Full Name"
-        />
-        <label>Email</label>
-        <input
-          className={classes.inputStyle}
-          type="email"
-          name="email"
-          defaultValue="Enter Your E-Mail Address"
-        />
-        <label className={classes.inputStyle}>I am a(n):</label>
+  const handleChange = (e) => {
+    setToSend({ ...toSend, [e.target.name]: e.target.value })
+  }
+
+  const SERVICE_ID = process.env.REACT_APP_SERVICE_ID
+  const TEMPLATE_ID = process.env.REACT_APP_TEMPLATE_ID
+  const USER_ID = process.env.REACT_APP_USER_ID
 
-        <select className={classes.inputStyle}>
-          <option>Please Select Who You Are</option>
-          <option>Prospective Volunteer</option>
-          <option>Prospective Benefactor</option>
-          <option>Attorney</option>
-          <option>Prosepective Partner</option>
-          <option>Other Individual/Organization</option>
-        </select>
-        <textarea
-          className={classes.inputStyle}
-          defaultValue="Type your message here."
-          type="text"
-          name="message"
-        />
-        <SecondaryButton text={"Send Message"} href="#" />
+  const handleFormSubmit = e => {
+    e.preventDefault();
+    send(
+      SERVICE_ID, TEMPLATE_ID, toSend, USER_ID
+    )
+    .then((response) => {
+      console.log('success', response.status, response.text)
+      alert("Email Sent Successfully")
+      // TODO: create custom pop up alert
+    })
+    .catch((error) => {
+      console.log('failed', error)
+      alert("ERROR. Please try again.")
+      // TODO: create custom pop up alert
+    })
+  };
+
+  const contactTypes = [
+    {
+      value: 'volunteer',
+      label: 'Prospective Volunteer',
+    },
+    {
+      value: 'benefactor',
+      label: 'Prospective Benefactor',
+    },
+    {
+      value: 'attorney',
+      label: 'Attorney',
+    },
+    {
+      value: 'partner',
+      label: 'Prospective Partner',
+    },
+    {
+      value: 'other',
+      label: 'Other Individual/Organization',
+    },
+  ];
+   
+  return (
+    <Grid
+      container
+      spacing={1}
+      direction="column"
+    >
+      <form onSubmit={handleFormSubmit}>
+        <FormControl  required  component="fieldset" className={classes.formStyle}>
+          <FormLabel className={classes.labelStyle} component="legend">Name</FormLabel>
+          <FormGroup>
+            <TextField 
+              id="standard-basic" 
+              className={classes.inputStyle}
+              label="Enter Your Full Name" 
+              required
```

Comment:
The required prompt does not line up quite right with the fields on empty submit.

Reply:
Do you mean the little dialog that comes up? If so, this is native to the browser, so I'm not sure if I can override the styes with CSS but I'll do some googlin

Reply:
Its not exactly the same thing, but [this ](https://v4.mui.com/components/text-fields/#validation) part of the api can be used for something similar, I think we would need to write our own validation conditions though. I didn't see anything in my research that it would be easy to adjust the required tag, since like you said it is browser native. I am happy with it as is, in this case.